docs/Getting-Started/Quickstart.md

```diff
@@ -0,0 +1,126 @@
+# Getting Started
+
+## Prequisites
+
+[Orion](../Installation/Install-Binaries.md)
+
+## Starting Orion and Sending a Payload
+
+To start Orion and send a payload: 
+1. Generate Keys 
+1. Create Password File
+1. Create a Configuration File 
+1. Start Orion 
+1. Confirm Orion is Running 
+1. Send a Payload
+1. Receive a Payload 
+
+### 1. Generate Keys
+
+To generate a public/private keypair for the Orion node:
+
+``` bash
+orion -g orion
```

Comment:
Instead of generating a key name orion, can't we use a different name? The command looks a bit confusing with two "orion" :)

Reply:
Fixed